examples/ex18p.cpp

```diff
@@ -1,24 +1,29 @@
-//                       MFEM Example 18 - Parallel Version
+//                                MFEM Example 18 - Parallel Version
 //
-// Compile with: make ex18
+// Compile with: make ex18p
 //
 // Sample runs:
 //
-//       mpirun -np 4 ex18p -p 1 -rs 2 -rp 1 -o 1 -s 3
-//       mpirun -np 4 ex18p -p 1 -rs 1 -rp 1 -o 3 -s 4
-//       mpirun -np 4 ex18p -p 1 -rs 1 -rp 1 -o 5 -s 6
-//       mpirun -np 4 ex18p -p 2 -rs 1 -rp 1 -o 1 -s 3
-//       mpirun -np 4 ex18p -p 2 -rs 1 -rp 1 -o 3 -s 3
+//       mpirun ex18p -p 1 -rs 2 -o 1 -s 3
+//       mpirun ex18p -p 1 -rs 1 -o 3 -s 4
+//       mpirun ex18p -p 1 -rs 0 -o 5 -s 6
+//       mpirun ex18p -p 2 -rs 1 -o 1 -s 3
+//       mpirun ex18p -p 2 -rs 0 -o 3 -s 3
 //
 // Description:  This example code solves the compressible Euler system of
 //               equations, a model nonlinear hyperbolic PDE, with a
-//               discontinuous Galerkin (DG) formulation.
+//               discontinuous Galerkin (DG) formulation in parallel.
+//
+//                (u_t, v)_T - (F(u), ∇ v)_T + <F̂(u,n), [[v]]>_F = 0
+//
+//               where (⋅,⋅)_T is volume integration, and <⋅,⋅>_F is face integration,
+//               F is the Euler flux function, and F̂ is the numerical flux.
 //
 //               Specifically, it solves for an exact solution of the equations
-//               whereby a vortex is transported by a uniform flow. Since all
+//               whereby a euler is transported by a uniform flow. Since all
```

Comment:
```suggestion
//               whereby a vortex is transported by a uniform flow. Since all
```
Looks like a find & replace typo here (and the one below)